Source/Extensions/Blazorise.Components/Autocomplete.razor

```diff
@@ -34,6 +34,8 @@
 
         @if ( DropdownVisible )
         {
+            JSRunner.RegisterClosableComponent( dotNetObjectRef, ElementRef );
```

Comment:
Just noticed this. Isn't it _dangerous_ to call JS async code in here? And it could potentially be called every time a component is rendered. We need to check for a boolean value jsRegistered to make sure it is registered only once. And possibly make use of `ExecuteAfterRender`.